tools/asan-ignore.txt

```diff
@@ -0,0 +1,13 @@
+[address]
+
+# from scipy.optimize._minpack._lmdif
+fun:enorm
+
+# from scipy.interpolate._dierckx.qr_reduce_periodic
```

Comment:
This one we just added, would be nice to investigate @czgdp1807 (does not block this PR of course).

Reply:
A first try for a low-hanging fruit would be to apply Sebastian's fix from CuPy , https://github.com/cupy/cupy/pull/9486

Short version: CuPy version of `deBoor_D` is a direct port from SciPY; the bug is a ~20 yr old use of unitialized memory; it never gave issues in SciPy but was causing segfaults in CuPy. I *think* Sebastian is right that in SciPy that unitialized memory is zero-initialized by `std::vector<double>`, but I did not gdb'd it myself.

Reply:
Will remove it tomorrow or on Monday. Will test on my fork first and then send a PR to the main SciPy repository.